### ISSUE TITLE
Empty events list (No events found)

### ISSUE THREAD

**Issue by kerf07** [jasonish/evebox#293]
I installed Evebox with sqlite on my server with suricata. it seems to run correct, without errors. 
<img width="1207" alt="Server_running" src="https://github.com/jasonish/evebox/assets/157745456/cb87466a-d948-46f6-ae98-d27341ae9b1c">
it also collect events id DB
<img width="358" alt="filesize" src="https://github.com/jasonish/evebox/assets/157745456/c5cc5234-2a8f-4ae0-a201-6f6b8b5b037b">
But doesn't show events in interface
<img width="1918" alt="empty_events" src="https://github.com/jasonish/evebox/assets/157745456/5c4ce35f-d2a6-49c5-9636-d576c2545d8c">
And i can't understand the reason

**Comment by kerf07** [comment]
i started evebox server with -vvv and, after refresh page in browser, saw "evebox::sqlite::eventrepo::alerts: Rows=0, Elapsed=1 ms" in logs

**Comment by jasonish** [comment]
This is very much how I use EveBox myself.  With `-vvv` on the agent and server you should see something like:

#### Agent
```
2024-02-03 17:14:14 TRACE evebox::agent::importer: Committing 3 events (bytes: 2693)
2024-02-03 17:14:14 TRACE evebox::bookmark: Writing bookmark /data/eb0e870c1998c30087f3be2ea20af6e0.bookmark
```

#### Server
```
2024-02-03 17:15:19 DEBUG evebox::sqlite::importer: Committed 3 events in 2 ms
```

Are you running the server and the agent on the same machine? If so, you could drop the agent and just modify the server unit file, or the server configuration file and have it consume the logs directly.

See `input` at https://evebox.org/docs/server/configuration

Might at least help with figuring out whats going on.

I assume the version is 0.17.2?

**Comment by kerf07** [comment]
Thank you for your frequent answer! And for advice to use only server (without agent).
While reading events.sqlite I mentioned that all records has "stats" type. And then I realized that for some reason my Suricata didn't detect any alerts.
Further research has shown that iptables rules, responsible for the work of IPS mode of Suricata, disappeared. I set them back, and working capacity of Suricata and Evebox has been restored.
